src/ComposableCoW.sol

```diff
@@ -130,19 +174,26 @@ contract ComposableCoW is ISafeSignatureVerifier {
         PayloadStruct memory _payload = abi.decode(payload, (PayloadStruct));
 
         // Check if the order is authorised
-        _auth(address(safe), _payload.params, _payload.proof);
+        bytes32 ctx = _auth(address(safe), _payload.params, _payload.proof);
```

Comment:
I didn't fully get what is `ctx` for. From what i can tell, is returning the hash that identifies the order, and only for `singleOrders`. But this is used later for `getTradeableOrder`. isn't it relevant for the Merkle root flow?
I guess I'm missing something